.github/workflows/publish-snapshot.yml

```diff
@@ -0,0 +1,26 @@
+name: check and publish snapshot Python tarball to PyPi
+
+on:
+    push:
+      branches-ignore:
+        - master
+        - version-*
+      paths-ignore:
+        - README.md
+
+jobs:
+  build-job:
+    name: Check and publish snapshot tarball to PyPi
+    uses: th2-net/.github/.github/workflows/compound-python.yml@python-workflow # FIXME: change to main
+    with:
+      release-type: development
+      style-check-enabled: false
+      strict-style-check: false
+      python-check-versions: "['3.8', '3.9', '3.10', '3.11', '3.12']"
+      test-dir: tests/tests_unit
+      test-ignore-dir: tests/tests_unit/test_dependencies/
+      test-requirements-file: requirements_dev.txt
+      scan-safety-args: '-i 70612'
```

Comment:
@davitmamrikishvili Why do we do scan twice? Here and in the .github/workflows/scanning.yaml ?

Reply:
As I understand, scanning.yaml will eventually go (when merge is completed in .github https://github.com/th2-net/.github/pull/91)

Reply:
@davitmamrikishvili 
Unfortunately, that doesn't answer the question.

What the reason to do the same check twice and duplicate `scan-safety-args: '-i 70612'` ?
This is probably a misstake.

Reply:
To be honest, I think it's safe to remove scanning.yaml right now too.

Reply:
I've just seen the explanation in the g-chat why we have it.
Solve the discussion.